twindle-web/index.html

```diff
@@ -9,7 +9,9 @@
   <body>
     <!-- Header Section Code for designing navigation -->
     <header>
-      <a href="/"><img src="./images/logo.svg" alt="Twitter Logo" /></a>
+      <a href="/" id="logo"><img src="./images/logo.svg" alt="Twitter Logo" /></a>
+    </header>
```

Comment:
You can't divide your header in two elements. Please, keep just one as original document.

Reply:
Thanks. I missed that part my bad.